src/c3chart/CommonND.js

```diff
@@ -133,6 +135,23 @@
             break;
         }
 
+        if(this.y2() instanceof Array && this.y2().length > 0){
```

Comment:
if y2 is of type array we dont need to check if its an array?  (though i know sometimes we set things null)